_posts/2022-03-10-introducing-openstreetmap-places.md

```diff
@@ -0,0 +1,94 @@
+   - archaeological (instead of archeological)
+   - chemist (instead of pharmacy)
+   - tyres (instead of tires)
+- There are two types of addresses provided in the data:
+   1. The address contained in the OSM data (`osm_addr`).  These were input by OSM contributors, but only a small percentage of the points come with an address (~24%).
+   1. The address of the nearest UGRC address point (`ugrc_addr`).  These are not official addresses of the businesses or points of interest, but provide a very close address for a greater percentage of points (~59%).  They are the nearest address (within 25m) in the [UGRC Address Points](https://opendata.gis.utah.gov/datasets/utah-address-points/explore) dataset.  The `addr_dist` field provides the distance (in meters) from the OSM point to the nearby address point.
+- The attributes can be used to filter the data down to your desired types of points
+   - The `category` field represents a general classification for a location.  Examples include: restaurant, fast_food, viewpoint, fire_station, etc.
+   - The other fields provide additional details about the location, where available
+      - `cuisine` - type of food available, may have multiple, separated by a ';' (american, italian, indian, burger, pizza, sandwich, ice_cream, etc.)
+      - `amenity` - often similar to the category field (restaurant, post_office, place_of_worship, social_facility)
+      - `shop` - type of shop (supermarket, toys, hardware, convenience, etc.)
+   - Some attributes might be duplicated among these fields
+- Additional information is provided in the other attributes, where available (`website`, `phone`, `open_hours`)
+- Several locations simply have a category of 'building' - these are prime for user-contributed improvements in [OpenStreetMap](https://www.openstreetmap.org/)! Stayed tuned for a future project to improve the `category` and attributes of these features…
+
+### What are the details on how this data is created?
+{: .text-left}
+
+The OSM Places layer is created by a Python script that pulls statewide OSM data from a nightly archive provided by [Geofabrik](https://www.geofabrik.de/data/download.html).  The archive data contains nearly 20 shapefiles, some that are relevant and some that aren't.  So, the OSM Places layer is built by filtering the data in those shapefiles down to a single point dataset with specific categories and attributes.  Additional de-duplication and spatial filtering are done along the way, to ensure the data is as clean as possible.  Then, additional fields are created and assigned from UGRC's SGID data (county, city, zip, nearby address, etc.) via point-in-polygon analysis.  Finally, additional attributes (OSM address, open hours, cuisine, etc.) are pulled from the [Overpass API](https://wiki.openstreetmap.org/wiki/Overpass_API) and joined to the filtered data using the 'osm_id' field as the join key.  Additional information can be found on the OpenStreetMap Places data page.
```

Comment:
**thought**: should this be a link?

> Additional information can be found on the OpenStreetMap Places data page

Reply:
Yes, but that link was breaking the netlify build because the page doesn't exist yet.  I'll add it after the data page gets merged.